backend/btrixcloud/operator/crawls.py

```diff
@@ -66,7 +66,7 @@
 
 
 # scale up if exceeded this threshold of mem usage (eg. 90%)
-MEM_SCALE_UP_THRESHOLD = 0.90
+MEM_SCALE_UP_THRESHOLD = 0.95
```

Comment:
Would maybe keep this at 90%? Otherwise it might be close enough to the max that we'll OOM before catching.

Reply:
oh yeah, didn't mean to keep that, the limit makes this unnecessary.